pkg/manifest/manifest_test.go

```diff
@@ -221,6 +221,19 @@ func TestManifest_Validate_Dockerfile(t *testing.T) {
 	assert.EqualError(t, err, "Dockerfile template cannot be named 'Dockerfile' because that is the filename generated during porter build")
 }
 
+func TestManifest_Validate_WrongSchema(t *testing.T) {
+	c := config.NewTestConfig(t)
+
+	c.TestContext.AddTestFile("testdata/porter-with-badschema.yaml", config.Name)
+	_, err := LoadManifestFrom(context.Background(), c.Config, config.Name)
+
+	assert.Error(t, err)
```

Comment:
nit - it feels like to me that if no error is returned here, the next assert would fail too. Is there a reason why we aren't using `require` here?